app/src/main/java/net/squanchy/tweets/util/TwitterDateUtils.java

```diff
@@ -0,0 +1,105 @@
+package net.squanchy.tweets.util;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.text.format.DateUtils;
+
+import com.twitter.sdk.android.core.models.Tweet;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+
+import net.squanchy.R;
+import net.squanchy.support.lang.Optional;
+
+public class TwitterDateUtils {
```

Comment:
I'd rather not risk this getting anywhere nearer to `develop` than this, if you don't mind doing it now, I'd prefer that